test/integration/eslint/test/index.test.js

```diff
@@ -24,25 +27,23 @@ const dirInvalidEslintVersion = join(__dirname, '../invalid-eslint-version')
 const dirMaxWarnings = join(__dirname, '../max-warnings')
 const dirEmptyDirectory = join(__dirname, '../empty-directory')
 const dirEslintIgnore = join(__dirname, '../eslint-ignore')
+const dirNoEslintPlugin = join(__dirname, '../no-eslint-plugin')
+const dirNoConfig = join(__dirname, '../no-config')
 
 describe('ESLint', () => {
   describe('Next Build', () => {
     test('first time setup', async () => {
       const eslintrc = join(dirFirstTimeSetup, '.eslintrc')
```

Comment:
Should this be `.eslintrc.json`?

Reply:
Doesn't need to be. This test updates an empty `.eslintrc` file regardless of format, but I changed it to `.eslintrc.json` anyway :)